test/executable_test.dart

```diff
@@ -73,6 +73,13 @@ void main() {
       expect(result.stderr, contains('dart_dev'));
     });
 
+    test('warns when the analyzer pacakge is ', () {
```

Comment:
I'm assuming this should be "does not warn when the analyzer dependency is used" and there should be a second additional test that is "warns when the analyzer dependency is unused"?